src/App.tsx

```diff
@@ -1,109 +1,33 @@
 import React from 'react';
-
 import './App.scss';
-
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+import { Comment } from './types/Comment';
+import { Post } from './types/Post';
+import { PostList } from './components/PostList';
+import { User } from './types/User';
+
+function getUserById(userId: number): User | null {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+function getCommentByPostId(postId: number): Comment[] {
+  const foundComment = commentsFromServer.filter(comment => (
```

Comment:
```suggestion
  return commentsFromServer.filter(comment => (
```
you can return immediately